build.gradle

```diff
@@ -138,6 +138,7 @@ dependencies {
     api group: 'org.opensearch', name:'opensearch-ml-client', version: "${opensearch_build}"
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.13.0'
     implementation "org.opensearch:common-utils:${common_utils_version}"
+    implementation "com.google.code.gson:gson:2.10.1"
```

Comment:
Haha! I knew I would get this comment from you. I used it for the parse method because ml-commons have it that way. I can fetch the method directly but I was trying to avoid the dependency on ml-commons by having our own method.

Reply:
https://github.com/opensearch-project/flow-framework/pull/198#discussion_r1410071989